src/fleet/azext_fleet/_help.py

```diff
@@ -23,6 +23,30 @@
           type: string
           short-summary: Prefix for hostnames that are created. If not specified, generate a hostname using the
                          managed cluster and resource group names.
+        - name: --enable-private-cluster
+          type: bool
+          short-summary: Prevents the Fleet's API server from be publicly accessed.
+        - name: --enable-vnet-integration
+          type: bool
+          short-summary: Enables the Fleet to communicate on a virtual network.
+        - name: --apiserver-subnet-id
+          type: string
+          short-summary: The sub network that the Fleet's API server will use.
```

Comment:
verbatim from swagger: 

"The subnet to be used when apiserver vnet integration is enabled. It is required when creating a new Fleet with BYO vnet."